src/test/java/org/apache/commons/jxpath/issues/JXPath113Test.java

```diff
@@ -54,21 +57,12 @@ public static Document getDocument(final InputSource is) throws Exception
             return builder.parse(is);
         }
 
-        private static DocumentBuilder getDocumentBuilder()
-        {
-            try
-            {
-                final DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
-                factory.setValidating(false);
-                factory.setNamespaceAware(true);
-                factory.setExpandEntityReferences(false);
-                return factory.newDocumentBuilder();
-            }
-            catch (final ParserConfigurationException e)
-            {
-                throw new Error("JAXP config error:" + e.getMessage(), e);
-            }
-
+        private static DocumentBuilder getDocumentBuilder() throws ParserConfigurationException {
```

Comment:
why not use `assertDoesNotThrow` here?

Reply:
Mainly because this is a "helper class" for the surrounding test. The implementation here must not care about assertions but give a clear idea of what is supposed to happen rather than testing behavior itself.

Reply:
I agree, the exception should just percolate up.